home/bin/ws.update.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use my127\Workspace\Application;
+use my127\Workspace\Updater\Exception\NoUpdateAvailableException;
+use my127\Workspace\Updater\Exception\NoVersionDeterminedException;
+use my127\Workspace\Updater\Updater;
+
+$pharPath = Phar::running(false);
+if (empty($pharPath)) {
+    echo 'This command can only be executed from within the ws utility.' . PHP_EOL;
+    exit(1);
+}
+
+$updater = new Updater('https://api.github.com/repos/my127/workspace/releases');
```

Comment:
is there a reason to do this outside of the app? could equally be a command?

Reply:
do what? it is a workspace command already, `ws self-update`

Reply:
i know, but why are doiing this in a PHP script?

Reply:
all the other commands sit in this directory too, but as bash scripts: https://github.com/my127/workspace/tree/0.2.x/home/bin - it was easier in PHP because you don't need to handle operating system differences in terms of available utilities, e.g. parsing the JSON becomes more difficult in bash when you don't know the host OS

Reply:
after discussing directly with you, I will move this to the Builder class where other commands are, in the future we can bring some consistency to the approach

Reply:
correction: not all commands are in this directory 😀 - I will move this to the builder

Reply:
unfortunately, the builder approach means the command is not listed when just executing `ws --help`:

```
./bin/workspace -h    

ws

Sub Commands:
   global        
   poweroff      Completely disable workspace by shutting down all containers
   create        
   secret        

Global Options:
  -h, --help       Show help message
  -v, --verbose    Increase verbosity
```

which is possibly why the other commands were added to `bin` directory as separate files, I will leave this as is for now for the same reason, it will need more work to bring consistency to the command approach whilst also providing useful output in the help